packages/wrangler/src/__tests__/tail.test.ts

```diff
@@ -598,9 +615,7 @@ function mockCreateTailRequest(
 		([_url, accountId, scriptName, envName], req) => {
 			requests.push(req);
 			expect(accountId).toEqual("some-account-id");
-			expect(scriptName).toEqual(
-				legacyEnv && env ? `test-worker-${env}` : "test-worker"
-			);
+			expect(scriptName).toEqual(expectedScriptName);
```

Comment:
Eventually, as we move towards MSW, we want to keep assertions out of our mocks -- but this is just a comment so you know and i know, not a thing that needs changing on this PR